components/FeaturedSlider.js

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import Link from "next/link";
+import Poster from "./Poster";
+
+// Import Swiper React components
+import { Swiper, SwiperSlide } from "swiper/react";
+
+// Import Swiper styles
+import "swiper/css";
+import "swiper/css/pagination";
+import "swiper/css/navigation";
+import "swiper/css/scrollbar";
+
+// import required modules
+import { Pagination } from "swiper";
+
+function FeaturedSlider({ gameProp, sliderTitle = "" }) {
+  console.log("---------Game Prop------");
+  console.log(gameProp[1].summary);
+
+  return (
+    <div>
+      <Swiper
+        loop={true}
+        pagination={{
+          dynamicBullets: true,
+        }}
+        style={{
```

Comment:
I haven't explored swiper that much, but is it possible to put these styles in the CSS? Like import the styles the same way we do with tailwind and define a class with these styles. If not dw about it.